CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+# 3.4.0 (2019-01-08)
+
+- Added invalid field status styles to textarea input. [#46](https://github.com/blackbaud/skyux-theme/pull/46)
```

Comment:
I'd change this to "Added style for invalid `textarea` elements."

FWIW, here's the reasoning for most of the changes:
- "invalid field status styles" is a difficult to process.
- It looks like we only added a single style, so "style" should be singular.
- Similarly, the new style applies collectively to all text area inputs, so "inputs" should be plural.
- We should either make "text area" two words or format it as a code snippet and use "elements" instead of "inputs."